addBag.php

```diff
@@ -1,16 +1,22 @@
+
+<?php
+    require_once ('getDbFunction.php');
+    require_once ('checkImageUrlFunction.php');
+    require_once ('checkEntriesFunction.php');
+    require_once ('addBagFunction.php');
+?>
```

Comment:
It doesn't affect anything functional, but you don't need to put your required files in brackets.